src/pages/media.js

```diff
@@ -1,4 +1,23 @@
 import React from 'react';
 import Layout from '../components/dev-hub/layout';
+import useAllVideos from '../hooks/use-all-videos';
+import { P } from '../components/dev-hub/text';
+import CardList from '../components/dev-hub/card-list';
+import usePodcasts from '../hooks/use-podcasts';
 
-export default () => <Layout></Layout>;
+export default () => {
+    const { videos, error, isLoading } = useAllVideos();
+    const {
+        podcasts,
+        error: errorPodcasts,
+        isLoading: isLoadingPodcasts,
+    } = usePodcasts();
+
+    return (
+        <Layout>
+            {isLoading && <P>Loading...</P>}
+            {videos && <CardList videos={videos} podcasts={podcasts} />}
+            {error && <P>Check back later for upcoming media contents</P>}
```

Comment:
```suggestion
            {error || errorPodcasts && <P>Check back later for upcoming media contents</P>}
```
This one is a bit trickier because we don't wanna show this if some videos or podcasts work, but let's revisit it later.